.storybook/data/tokens.json

```diff
@@ -80,6 +80,7 @@
   "eds-size-8": "4rem",
   "eds-size-9": "4.5rem",
   "eds-size-10": "5rem",
+  "eds-size-12": "6rem",
```

Comment:
This feels weird and I think requires some discussion with design on if we can compromise on spacing between the top `Panel` and the content below.

Reply:
Sounds good, will remove for now